packages/core/src/sagas/effects/observeOnce.js

```diff
@@ -8,13 +8,11 @@ export default function observeOnceEffect(observable) {
       return yield call(
         () =>
           new Promise((resolve, reject) => {
-            observable.subscribe(resolve, reject);
+            subscription = observable.subscribe(resolve, reject, resolve);
```

Comment:
Can you explain why next and complete are the same? to me it seems like this would make resolve be called twice. :/

Reply:
`observeOnce` is essentially converting a stream into an async binary operation (success or fail in async fashion, more-or-less Promise).

For "complete without any results", I think it should be considered as "positive with result of `undefined`".